.env

```diff
@@ -22,6 +22,8 @@ IDAM_KEY_BULK_SCAN_PROCESSOR=AAAAAAAAAAAAAAAA
 IDAM_KEY_BULK_SCAN_ORCHESTRATOR=AAAAAAAAAAAAAAAA
 IDAM_KEY_XUI_WEBAPP=AAAAAAAAAAAAAAAA
 IDAM_KEY_AM_ROLE_ASSIGNMENT=AAAAAAAAAAAAAAAA
+IDAM_KEY_TS_TRANSLATION_SERVICE=AAAAAAAAAAAAAAAC
+TS_TRANSLATION_SERVICE_IDAM_CLIENT_SECRET=AAAAAAAAAAAAAAAD
```

Comment:
I don't think translation service will be using an IDAM client: so TS_TRANSLATION_SERVICE_IDAM_CLIENT_SECRET can probably be removed.